.github/workflows/django.yml

```diff
@@ -44,7 +44,5 @@ jobs:
         run: poetry install
         
       - name: Run Django tests
-        working-directory: ./src
-        run: poetry run python manage.py test
-        env:
-          PATH: "$HOME/.local/bin:$PATH"
+        shell: bash
+        run: poetry run python src/manage.py test
```

Comment:
We need to use `working-directory: ./src` in order for `manage.py` to autodiscover tests

Reply:
Fixed, just specified the working-directory while keeping the bash shell explicitly defined.